scenarios/4p_A_New_Land.cfg

```diff
@@ -437,24 +504,30 @@
         [filter]
             side=5,6,7,8
             [filter_location]
-                terrain=Re^Gvs
+                terrain=*^Gvs
             [/filter_location]
         [/filter]
         {MODIFY_TERRAIN "Rd" $x1 $y1}
         {PLACE_IMAGE "scenery/castle-ruins.png" $x1 $y1}
+        # FIXME: there exist two more variations of that image, use them (see TSG S2).
         [sound]
             name=pincers.ogg
         [/sound]
     [/event]
 
-    # Universities & Castles
+    # Castles
     [event]
         name=moveto
         first_time_only=no
         [filter]
             side=5,6,7,8
+            canrecruit=no # workaround
             [filter_location]
-                terrain=C*,K*
+                terrain=C*^*,K*^* # FIXME: This causes AI units to destroy their own castles too.
```

Comment:
Do you mean that the ai units in the keep will sabotage their own keep if there are player units blocking them from moving further? I feel this would make it too easy. Or is this already in the game, and it's not really as big a of a deal as I think?